src/xgov-dapp/src/features/vote-creation/review/index.tsx

```diff
@@ -0,0 +1,203 @@
+import { Alert, Button, Link, Typography } from '@mui/material'
+
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import api from '../../../shared/api'
+import { LoadingDialog } from '../../../shared/loading/LoadingDialog'
+import {
+  VoteCreationReviewSteps,
+  useAppReference,
+  useAuth,
+  useResetCreateRound,
+  useReviewStep,
+  useRoundInfo,
+  useSetAppReference,
+  useSetAppSourceMaps,
+  useSetAuth,
+  useSetReviewStep,
+} from '../state'
+import { ConfirmationDialog } from './ConfirmationDialog'
+
+import { useWallet } from '@txnlab/use-wallet'
+import Papa from 'papaparse'
+import { ProposalCard } from '../../../shared/ProposalCard'
+import { VotingRoundGlobalState } from '../../../shared/VotingRoundContract'
+import { Proposal } from '../../../shared/types'
+import { VotingTime } from '../../vote/VotingTime'
+
+export default function Review() {
+  const [confirmationDialogOpen, setConfirmationDialogOpen] = useState(false)
+  const { activeAddress, signer: transactionSigner } = useWallet()
+  const roundInfo = useRoundInfo()
+  const proposalsParsed = Papa.parse<Proposal>(roundInfo.proposalFile, { header: true }).data
```

Comment:
What happens if the csv is corrupt, do we handle it nicely?

Reply:
There's pretty full on validation on the previous screen but yeah I can make it more defensive here too. Will address in a separate PR though